pgraph/graphsync_test.go

```diff
@@ -92,3 +92,41 @@ func TestGraphSync2(t *testing.T) {
 		t.Errorf("%s", s)
 	}
 }
+
+func TestGraphSync3(t *testing.T) {
+	v1 := NV("v1")
+	v2 := NV("v2")
+	v3 := NV("v3")
+
+	e1 := NE("e1")
+	e2 := NE("e2")
+	e3 := NE("e3")
+	e4 := NE("e4")
+
+	//Base Graph with 3 edges
```

Comment:
comments should be full sentences which start with a capital and end with a period, or not full sentences and not be capitalized. in both case the syntax is `// hello` not `//hello` (add a space after the `//`)